tests/draft-next/additionalProperties.json

```diff
@@ -152,5 +152,64 @@
                 "valid": true
             }
         ]
+    },
+    {
+        "description": "propertyDependencies with additionalProperties",
+        "schema": {
+            "$schema": "https://json-schema.org/draft/next/schema",
+            "propertyDependencies": {
+                "foo": {
+                    "bar": {
+                        "properties": {
+                            "buz": {}
+                        }
+                    }
+                }
+            },
+            "additionalProperties": false
+        },
+        "tests": [
+            {
+                "description": "additionalProperties doesn't consider propertyDependencies properties" , 
+                "data": {"foo": "bar"},
+                "valid": false
+            },
+            {
+                "description": "additionalProperties doesn't consider the innner schema properties",
+                "data": {"foo": "bar", "buz": ""},
+                "valid": false
+            }
+        ]
+    },
+    {
+        "description": "dependentSchemas with additionalProperties",
+        "schema": {
+            "$schema": "https://json-schema.org/draft/next/schema",
+            "dependentSchemas": {
+                "foo": {
+                    "properties": {
+                        "bar":{}
+                    }
+                }
+            },
+            "additionalProperties": false
+        },
+        "tests": [
+            {
+                "description": "additionalProperties doesn't consider dependentSchemas",
+                "data": {"foo": ""},
+                "valid": false
+            },
+            {
+                "description": "additionalProperties does't consider inner schema property",
+                "data": {"bar": ""},
+                "valid": false
+            },
+            {
+                "description": "additionalProperties doesn't consider both",
+                "data": {"bar": "", "foo": ""},
+                "valid": false
+            }
```

Comment:
Define `foo`.  This will still be invalid since `additionalProperties` can't see the subschema that defines `bar`.